helm_remote/Tiltfile

```diff
@@ -178,7 +181,21 @@ def helm_remote(chart, repo_url='', repo_name='', release_name='', values=[], se
     # TODO: since neither `k8s_yaml()` nor `helm()` accept resource_deps,
     # sometimes the crds haven't yet finished installing before the below tries
     # to run
-    yaml = helm(chart_target, name=release_name, namespace=namespace, values=values, set=set)
+    helm_tpl_cmd = ['template', release_name, chart_target, '--include-crds', '--namespace', namespace]
+
+    if len(values) != 0:
+        for v in values:
+            helm_tpl_cmd.extend(['--values', v])
+
+    if len(set) != 0:
+        for s in set:
+            helm_tpl_cmd.extend(['--set', s])
+
+    if validate:
+        helm_tpl_cmd.append('--validate')
+
+
+    yaml = local(build_helm_command(helm_tpl_cmd, None, None))
```

Comment:
i'm not sure that `build_helm_command` really adds much if you're not using the other args :joy: